docs/guides/getting-started.md

```
@@ -3,25 +3,48 @@
     ```
+
 4. Open [http://localhost:8000](http://localhost:8000) in your browser.
+
+## Using Docker
+
+```bash
+# Build and start containers
+docker-compose up --build
+```
+
+**For advanced usage go to [advanced-usage.md](advanced-usage.md)**
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)